test/integration/vaultdynamicsecret_integration_test.go

```diff
@@ -190,7 +190,17 @@ func TestVaultDynamicSecret(t *testing.T) {
 		existing           int
 	}{
 		{
-			name:     "existing-only",
+			name:           "one-existing-only",
```

Comment:
Could we maybe set `withArgoRollout` on the `create-only` test below? Is there a problem running more than one test with argo rollouts?

Reply:
I've removed `one-existing-only` test case and added `withArgoRollout: true` to both `existing-only` and `create-only` test cases. Tests passed locally.

Reply:
Looks like CI workflow has problems with running more than one test with argo rollouts. Is it ok to have `withArgoRollout` in one test case?